src/test/resources/feature/3-destinatario/personaFisica/ricercaNotifica/209_A209_visualizzazioneDettaglioNotificaInseritaConAPIV1.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Ricerca notifica per periodo temporale persona fisica
+
+  @TestSuite
+  @TA_PFVisualizzaNotificaApiV2
+  @PF
+  @PFRicercaNotifica
+
+  Scenario: PN-9441 - Visualizzazione dettaglio notifica inserita con API v.2
+    Given Login Page persona fisica test viene visualizzata
+    When Login con persona fisica
+      | user         | cristoforocolombo |
+      | pwd          | password123       |
+      | name         | Cristoforo        |
+      | familyName   | Colombo           |
+      | fiscalNumber | TINIT-CLMCST42R12D969Z  |
+    And Si visualizza correttamente la pagina Piattaforma Notifiche persona fisica
+    And Nella pagina Piattaforma Notifiche  persona fisica inserire il codice IUN "MXVM-JQTG-MQZW-202312-W-1"
```

Comment:
as you know it's the right one

Reply:
Codice IUN is provided in the test case and mapping.